src/app/page/page.component.ts

```diff
@@ -746,6 +755,17 @@ export class PageComponent implements OnInit, OnDestroy {
       }
     });
 
+    const manifestDismissListeners =
+      this._pageBookManifest?.dismissListeners || [];
+    const isManifestDismissListener = manifestDismissListeners.some(
+      (listener) => listener.name === event
+    );
+
+    if (!pageToNavigateTo && isManifestDismissListener) {
```

Comment:
I don't think you would need to keep the check on `pageToNavigateTo` here, closing a tool would have a higher priority than switching pages. In reality the [spec documentation](https://github.com/CruGlobal/mobile-content-api/blob/master/public/xmlns/manifest.xsd#L47-L49) recommends to fire off multiple events instead of relying on the same event to trigger multiple listeners

> Because multiple events can be triggered in sequence by any element that triggers events, it makes sense to prefer triggering multiple subsequent events when resolution of a single event to multiple listeners would give ambiguous results.